test/functional/test-resources.js

```diff
@@ -966,9 +967,13 @@ describe('Resources.Resource', () => {
 
     resources = new Resources(window);
     resource = new Resource(1, element, resources);
+    viewportMock = sandbox.mock(resources.viewport_);
   });
 
   afterEach(() => {
+    viewportMock.verify();
```

Comment:
What I found out recently is that if you use `sandbox.mock` you don't need to call verify/restore - sandbox will do that.